docs/topics/tour/kotlin-tour-hello-world.md

```diff
@@ -0,0 +1,115 @@
+[//]: # (title: Hello world)
+
+<microformat>
+    <p>This is the first part of the Kotlin tour:</p>
+    <p><img src="icon-1.svg" width="20" alt="First step" /> <strong>Hello world</strong><br />
+        <img src="icon-2-todo.svg" width="20" alt="Second step" /> <a href="kotlin-tour-basic-types.md">Basic types</a><br />
+        <img src="icon-3-todo.svg" width="20" alt="Third step" /> <a href="kotlin-tour-collections.md">Collections</a><br />
+        <img src="icon-4-todo.svg" width="20" alt="Fourth step" /> <a href="kotlin-tour-control-flow.md">Control flow</a><br />
+        <img src="icon-5-todo.svg" width="20" alt="Fifth step" /> <a href="kotlin-tour-functions.md">Functions</a><br />
+        <img src="icon-6-todo.svg" width="20" alt="Sixth step" /> <a href="kotlin-tour-classes-part-1.md">Classes</a><br />
+        <img src="icon-7-todo.svg" width="20" alt="Final step" /> <a href="kotlin-tour-null-safety.md">Null safety</a></p>
+</microformat>
+
+Below is a simple program that prints "Hello, world!":
+
+```kotlin
+fun main() {
+    println("Hello, world!")
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="hello-world-kotlin"}
+
+In Kotlin:
+* `fun` is used to declare a function
+* the entry point is the `main` function
+* function arguments are written within parentheses `()`
```

Comment:
I decided to remove the mention of arguments completely. It will be covered later in the Functions chapter and it isn't necessary to introduce it here.